### DISCUSSION TITLE
Doppler filters

### DISCUSSION THREAD

**Discussion by pfeatherstone** [jgaeddert/liquid-dsp#329] [General]
Do you have a reference for `liquid_firdes_doppler()` at https://github.com/jgaeddert/liquid-dsp/blob/8bf87b6fe325d98c250d6911fa50518d14175d86/src/filter/src/firdes.c#L454 ?

I'm interested in fading models and all the possible ways they can be simulated. Hard to find some good resources that aren't buried in a $200 text book written 30 years ago.

**Comment by seonlee2013** [comment]
好啊，邮件收到啦！
Hi, your email is arrived safely

**Comment by pfeatherstone** [comment]
Is it likely to be somewhere in "Simulations of Communication systems" by Jeruchim, Balaban, Shanmugan ?
